cypress.config.ts

```diff
@@ -6,6 +6,7 @@ export default defineConfig({
 	defaultCommandTimeout: 8000,
 	responseTimeout: 8000,
 	requestTimeout: 8000,
+	numTestsKeptInMemory: 1,
```

Comment:
This is in here in the hope that it might help with the running of the cypress tests. When running the tests locally I was occasionally getting an out of memory error. Lowering this value from it default of 50 to 1 or 0 was one suggestion to help.